provision-base.sh

```diff
@@ -10,12 +10,20 @@ pandora_ip_address="${1:-10.10.0.2}"; shift || true
 #     dpkg-preconfigure: unable to re-open stdin: No such file or directory
 export DEBIAN_FRONTEND=noninteractive
 
+# Regenerate Machine UUID
```

Comment:
please drop this. my base images are already created without a machine-id (this means it will be generated in their first boot).